example_app/web/router.ex

```diff
@@ -18,10 +19,9 @@ defmodule ExampleApp.Router do
     get "/show", ExampleApp.PageController, :show
     get "/forbidden", ExampleApp.PageController, :forbidden
 
-    post "/register", Addict.Controller, :register
-    post "/logout", ExampleApp.UserController, :logout
-    post "/login", Addict.Controller, :login
-    post "/password_recover", Addict.Controller, :recover_password
-    post "/password_reset", Addict.Controller, :reset_password
+    addict :routes,
+      logout: [path: "/logout", controller: ExampleApp.UserController, action: :signout],
+      recover_password: "/password/recover",
+      reset_password: "/password/reset"
```

Comment:
Can you also update [`index.html.eex`](https://github.com/trenpixster/addict/blob/master/example_app/web/templates/page/index.html.eex) with the new routes? :smile: